PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.c

```diff
@@ -305,7 +305,11 @@ PcRtcInit (
   //
   // Clear RTC register D
   //
-  RegisterD.Data = FixedPcdGet8 (PcdInitialValueRtcRegisterD);
+  // On most platforms VRT is read-only, but when it's not it must
+  // be preserved to prevent failing in RtcWaitToUpdate().
+  //
+  RegisterD.Data = RtcRead (RTC_ADDRESS_REGISTER_D);
+  RegisterD.Data = FixedPcdGet8 (PcdInitialValueRtcRegisterD) | (RegisterD.Data & BIT7);

```

Comment:
If BIT7 is set in PCD, BIT7 will be set no matter what state RegisterD is in.  Is this the expected behavior?

Reply:
Not sure if or when BIT7 should be set. Updated the commit to not set BIT7 by PcdInitialValueRtcRegisterD.